Content.Server/Singularity/EntitySystems/RadiationCollectorSystem.cs

```diff
@@ -36,7 +56,32 @@ private void OnInteractHand(EntityUid uid, RadiationCollectorComponent component
 
         private void OnRadiation(EntityUid uid, RadiationCollectorComponent component, OnIrradiatedEvent args)
         {
-            if (!component.Enabled) return;
+            if (!component.Enabled || component.RadiationReactiveGases == null)
+                return;
+
+            if (!TryGetLoadedGasTank(uid, out var gasTankComponent) || gasTankComponent == null)
+                return;
+
+            var charge = 0f;
+
+            foreach (var gas in component.RadiationReactiveGases)
+            {
+                float reactantMol = gasTankComponent.Air.GetMoles(gas.Reactant);
+                float delta = args.TotalRads * reactantMol * gas.ReactantBreakdownRate;
+
+                float temperatureMod = 1.5f * gasTankComponent.Air.Temperature / (150f + gasTankComponent.Air.Temperature);
```

Comment:
Comment describing the rough intent of this temperatureMod and how it was designed would be welcome